### PR TITLE
Add Ukrainian localization for Mod Menu Helper

### PR DESCRIPTION
**Pull request by Den4enko**

### CODE REVIEW

File: Mod Menu Helper/assets/fo/lang/uk_ua.json [hunk 1]
```diff
@@ -0,0 +1,58 @@
+{
+  "modmenu.dropInfo.line1": "У FO модів є §9✎§r або §8✎§r, альтернативи як §c*§r;",
+  "modmenu.dropInfo.line2": "інші не підтримуються",
+  "modmenu.summaryTranslation.advancementinfo": "Більший, розумніший екран досягнень\n§9✎§8 Кольори, розмір вікна інформації, ...",
+  "modmenu.summaryTranslation.advancements-enlarger": "§c*§r Більший екран досягнень\n§8✎ (без налаштувань)",
+  "modmenu.summaryTranslation.animatica": "Анімовані текстури OptiFine\n§8✎ (лише продвинуті налаштування)",
+  "modmenu.summaryTranslation.antighost": "Виправлення блоків-привидів\n§9✎§8 Клавіши",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/uk_ua.json:7]:
```suggestion
  "modmenu.summaryTranslation.antighost": "Введіть /ghost, щоб виправити блоки-привиди\n§9✎§8 Клавіши",
```
The first line should be identical to the next string and I confirmed that it fits

File: Mod Menu Helper/assets/fo/lang/uk_ua.json [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  "isxander-main-menu-credits.antighost": "Введіть /ghost, щоб виправити блоки-привиди",
+  "modmenu.summaryTranslation.betterbeds": "§c*§r Оптимізація ліжок\n§8✎ (без налаштувань)",
+  "modmenu.summaryTranslation.bettermounthud": "Показувати їжу та XP на коні\n§8✎ (без налаштувань)",
+  "modmenu.summaryTranslation.bobby": "§c*§r Зберігання завантажених чанків видимимі\n§9✎§8 Блокування сутностей, приглушити чанки, зберігати чанки, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/uk_ua.json:11]: Probably will be cut off after `чанки,`, so you could just remove the rest.

File: Mod Menu Helper/assets/fo/lang/uk_ua.json [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+  "modmenu.summaryTranslation.borderlessmining": "Повноекранний режим з Alt+Tab (опц.)\n§9✎§8 Перемикання моду, Підтримка macOS, індивідуальний розмір, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/uk_ua.json:12]: The string is cut off after `macOS,`, so you could just remove the rest.

File: Mod Menu Helper/assets/fo/lang/uk_ua.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+  "modmenu.summaryTranslation.capes": "Плащі OptiFine та інші\n§9✎§8 Типи, переважний тип, Елітри, ...",
+  "modmenu.summaryTranslation.citresewn": "Користувацькі предмети OptiFine\n§9✎§8 Перемикання моду, дозволити зламані шляхи, ...",
+  "modmenu.summaryTranslation.colormatic": "Користувацькі кольори OptiFine\n§9✎§8 Чисті небо/порожнеча, згладжування освітлення",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/uk_ua.json:15]: String is cut off after `згладжування`

File: Mod Menu Helper/assets/fo/lang/uk_ua.json [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+  "modmenu.summaryTranslation.cem": "Користувацькі сутності OptiFine (альфа)\n§9✎§8 Підтримка OptiFine, виправлення моделей",
+  "modmenu.summaryTranslation.debugify": "Виправлення помилок, відключення діагностики\n§9✎§8 Дивіться у вікі FO",
+  "modmenu.summaryTranslation.dynamicfps": "Зменшення FPS у фоновому режимі\n§9✎§8 Перемикання моду, відновлення при наведенні, ...",
+  "modmenu.summaryTranslation.enhancedblockentities": "Оптимізація скринь, табличкі, ліжка т.д.\n§9✎§8 Блоки для покращення, метод",
+  "modmenu.summaryTranslation.entity_texture_features": "Випадкові й світлові моби OptiFine\n§9✎§8 Випадкові моби, сяяння, кліпання очей, ...",
+  "modmenu.summaryTranslation.entityculling": "Припинення рендеру сутностей за стінами\n§8✎ (лише продвинуті налаштування)",
+  "modmenu.summaryTranslation.fabricskyboxes": "Настроюване небо\n§9✎§8 Розгорніть мод для налаштування (клік ▼)",
+  "modmenu.summaryTranslation.fsb-interop": "Підтримка OptiFine для FabricSkyBoxes\n§9✎§8 OptiFine/MCPatcher/нативна підтримка, ...",
+  "modmenu.summaryTranslation.fabrishot": "Створення HD скріншотів за натисканням F9\n§9✎§8 Перепризначення F2, роздільна здатність, клавіши, ...",
```
Comment (Madis0) [inline Mod Menu Helper/assets/fo/lang/uk_ua.json:26]: String is cut off after `здатність,`, so you can remove the rest

**Review by Madis0 — CHANGES_REQUESTED** [review]
Cool! Needs some small tweaks though, you may also find https://fabulously-optimized.github.io/Mod-Menu-Helper-Size-Checker/ helpful.

File: Mod Menu Helper/assets/fo/lang/uk_ua.json (hunk 4 above)
Comment (Den4enko) [reply Mod Menu Helper/assets/fo/lang/uk_ua.json]: Just used synonym verb

**Comment by Madis0** [comment]
Thank you!

**Pull request merged**